readme.md

```diff
@@ -104,7 +104,6 @@ Various tools for working with images such as cropping tools.
 
 - [Leaflet-IIIF Cropping](https://bl.ocks.org/mejackreed/6936585f435b60aa9451ae2bc1c199f2) - Example of using Leaflet to provide IIIF cropping.
 - [Stanford Cropper](https://github.com/lizfischer/iiif-tools#cropper) - Simple image cropper.
-- [OpenSeadragon Cropping Tool](http://sul-dlss.github.io/iiif-cropper/demo/) - Script to allow for cropping an image from within OpenSeadragon.
```

Comment:
The url is now here: http://sul-dlss-deprecated.github.io/iiif-cropper/demo/

I would have thought GitHub would have redirected that automatically :(

Reply:
Ah, okay. I've updated the link instead

Reply:
Could you confirm this is okay @mejackreed ?